EnglishTranslation-master/QUEST_LV_0300.tsv

```diff
@@ -1206,297 +1206,297 @@ QUEST_LV_0300_20160310_001205	What do we do with the useless scriptures?{nl}We u
+QUEST_LV_0300_20160310_001390	You found them all? {nl}I guess I can be relieved now.
+QUEST_LV_0300_20160310_001391	Hmm, maybe... It's just an idea.{nl}How about we throw the powder that came from the pocket to the far place from the pit.{nl}
 QUEST_LV_0300_20160310_001392	Then, the monsters will be directed there. Right?{nl}Can you help us out?{nl}
 QUEST_LV_0300_20160310_001393	Near the Atskiles Cay would be the ideal spot.{nl}Godspeed.
-QUEST_LV_0300_20160310_001394	It's embarassing as an elder, but I once thought to give up on everything.{nl}Whatever it is, I think we made the right decision to stand still. 
-QUEST_LV_0300_20160310_001395	An enormous number of monsters came? {nl}Damn.. effects seem good, but we will be in a danger.{nl}
-QUEST_LV_0300_20160310_001396	I am so sorry to give you a hard task.{nl}I will tell other people not to touch on the pocket.{nl}
-QUEST_LV_0300_20160310_001397	That coral.. Where did you get it?{nl}That's called Aru. That is a very important coral that is being used in medicine.
-QUEST_LV_0300_20160310_001398	There must be it's habitat near the place where you obtained it..{nl}
-QUEST_LV_0300_20160310_001399	I am sorry, but can you look for their habitat?{nl}If you could look for the habitat of corals, it would help the villagers a lot.
-QUEST_LV_0300_20160310_001400	If you could find the habitat of the corals of Aru, it will help the villagers a lot.
-QUEST_LV_0300_20160310_001401	Nugara Sand Saw?{nl}That deep�� we will have hard time extracting it.{nl}
-QUEST_LV_0300_20160310_001402	First, we better let the refugees know it.{nl}We better look for a way together.{nl}Thank you so much!
-QUEST_LV_0300_20160310_001403	I am sorry, but can you help us out?{nl}We have many wounded people, but the medicine ran out.. It would become more severe if we leave it like this..{nl}
-QUEST_LV_0300_20160310_001404	We've already used all the medicine which the merchants had and we have no way of getting the medicine again.{nl}We can make the ointment if we have the ingredients..
-QUEST_LV_0300_20160310_001405	Please, get me Greentoshell Shell Fragments and Blue Lapasape Mosses.{nl}With them, I can make emergency ointment for the injuries.
-QUEST_LV_0300_20160310_001406	It's not a severe cut, but if we leave it, it will decay.{nl}There are many people hurt, but I can't do anything..
-QUEST_LV_0300_20160310_001407	Thank you. Thank you so much.{nl}This would help you a lot.
-QUEST_LV_0300_20160310_001408	I am sorry. I found a small box that was dropped on the wrecked ship..{nl}Do you know anything about it?{nl}
-QUEST_LV_0300_20160310_001409	Ahah.. So you don't know.{nl}This is sealed with magic so it can't be opened.{nl}
-QUEST_LV_0300_20160310_001410	Some important object seems to be inside..{nl}Can you help me opening the box?
+QUEST_LV_0300_20160310_001394	It's embarassing as mayor, but I once thought to give up on everything.{nl}Whatever it is, I think we made the right decision to stand still. 
```

Comment:
"As mayor, it's embarrassing, but I almost gave up on everything.{nl}Whatever it is, I think we made the right choice with staying put."